anemometer/app/entrypoint.sh

```diff
@@ -0,0 +1 @@
+node src/price-feed.js $@
```

Comment:
should start with `#!/bin/bash`

Reply:
also, src/price-feed.js should be renamed

Reply:
renamed all the files to new names that make sense according to this project. 

Just because I missed this first line in my entrypoint.sh my application was producing a file (which wasn't even downloading) and not encoded output. However, #!/bin/sha is the correct code that needs to be used. Thanks for pointing it out.